extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -51,10 +51,12 @@ class AdNetworkConfigDef {
   getAttributes() {}
 
   /**
-   * Network specific constraints on the placement of ads on the page.
+   * Network specific constraints on the placement of ads on the page. Used
+   * when no ad constraints are specified in the over-the-wire config supplied
+   * by the ad network.
    * @return {!./ad-tracker.AdConstraints}
    */
-  getAdConstraints() {}
+  getDefaultAdConstraints() {}
```

Comment:
Can we keep the `getAdConstraints` method, and only add the `getDefaultAdConstraints` to `AdSenseNetworkConfig`. Then we can still call `adNetwork.getAdConstraints()` in `amp-auto-ads.js`. 
This doesn't make much difference now since adsense is the only supported ad network. But other ad networks might not want to support the ad constraints config when they add support, I think `getAdConstraints` is a general enough method for ad networks to overwrite.